problem_builder/dashboard.py

```diff
@@ -238,10 +238,26 @@ class DashboardBlock(StudioEditableXBlockMixin, XBlock):
         help=_("Toggles if numeric values are displayed"),
         scope=Scope.content
     )
+    header_html = String(
+        display_name=_("Header HTML"),
+        default="",
+        help=_("Additional HTML to include after the heading."),
+        multiline_editor="html",
+        resettable_editor=False,
+        scope=Scope.content,
+    )
+    footer_html = String(
+        display_name=_("Footer HTML"),
+        default="",
+        help=_("Additional HTML to include after the bottom of the page."),
```

Comment:
Similarly this is confusing, I would say,

"HTML to include after the problem." (or question)

In both of these it occurs to me that "display" may be better than "include"